### DISCUSSION TITLE
Encountering `No help topic for chain` using `lncli chain getblockheader` command

### DISCUSSION THREAD

**Discussion by mohamedawnallah** [lightningnetwork/lnd#8459] [Q&A]
**Description:**
Apologies for the dump question, but I'm encountering an issue where I receive a `No help topic for {command}` message, even though the `{command}` in question is present in the Shell code sample provided in the Lightning Engineering documentation. For instance, when using a command like `lncli chain getblockheader`.

**Referenced Documentation:**
[Lightning Engineering API Docs - Get Block Header](https://lightning.engineering/api-docs/api/lnd/chain-kit/get-block-header)

**Additional Context:**
I've compiled lnd from the source

**Comment by guggero** [comment]
You need to include the proper build tags. E.g. `make install tags="chainrpc"`. Or just use the `make release-install` command that includes all build tags the release binary would contain.

**Reply by guggero** [reply]
Also see the "Tags" section in https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#installing-lnd-from-source.

**Reply by mohamedawnallah** [reply]
Thanks! I have already followed the tags section in the docs, but I've just found the issue. I created aliases for `lnd-debug` and `lncli-debug` to use shorter commands. However, `lnd-debug` and `lncli-debug` were not getting updated when executing `make install tags="..."` and it is supposed to since it's meant for release purposes. I resolved the issue by removing the aliases, and now it works 👍.
